localstack/services/kinesis/kinesis_listener.py

```diff
@@ -165,9 +165,10 @@ def return_response(self, method, path, data, headers, response):
         elif action == "PutRecord":
             response_body = self.decode_content(response.content)
             # Note: avoid adding 'encryptionType':'NONE' in the event_record, as this breaks .NET Lambdas
+            # also, ensure data payload is base64 encoded before passing to lambda functions
             event_record = {
                 "approximateArrivalTimestamp": epoch_timestamp(),
-                "data": data["Data"],
+                "data": to_str(base64.b64encode(data["Data"])),
```

Comment:
Thanks for updating the @KOConchobhair .

Seeing this in the CI build logs now:
```
TypeError: a bytes-like object is required, not 'str'
```

Can we please wrap the bytes to be encoded via our `to_bytes()` util function:
```
                 "data": to_str(base64.b64encode(to_bytes(data["Data"]))),
```

Same for line 190 below.

Reply:
okay, fixed and rebased the latest from master to remove merge commit. 
I did not get this error when verifying this fix outside the unit test, so I think it may be testing some additional cases with strings or something.